### PR TITLE
Add DB API support

### PR DESCRIPTION
**Pull request by johnsudaar**

### CODE REVIEW

File: addons.go [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 	AddonDestroy(app, addonID string) error
 	AddonUpgrade(app, addonID, planID string) (AddonRes, error)
+	AddonToken(app, addonID string) (string, error)
 }
 
 var _ AddonsService = (*Client)(nil)
 
 type Addon struct {
 	ID              string         `json:"id"`
+	UUID            string         `json:"uuid"`
```
Comment (Soulou) [inline addons.go]: The API is not returning the UUID as ID?

Comment (johnsudaar) [reply addons.go]: Nop

Comment (Soulou) [reply addons.go]: Wouldn't it be the way to go?

Comment (johnsudaar) [reply addons.go]: Maybe but it's not really in the scope of this PR.

Comment (Soulou) [reply addons.go]: But adding it now to remove it later (and breaking the API) is not the greatest thing to do no?

Comment (Soulou) [reply addons.go]: https://github.com/Scalingo/api/pull/1342

**Pull request merged**